convert2rhel/subscription.py

```diff
@@ -201,8 +201,12 @@ def _stop_rhsm():
     """Stop the rhsm service."""
     cmd = ["/bin/systemctl", "stop", "rhsm"]
     if system_info.version.major <= 6:
-        # RHEL6 used upstart.  After that, we use systemd
-        cmd = ["service", "rhsm", "stop"]
+        # On RHEL6, there isn't a service-oriented way to stop rhsm.  It is started on demand so
+        # there isn't an init script to stop it.  If we find we need to stop it, because we're
+        # etting "machine is already registered" errors there, then we'll need to look for
+        # rhsm-service in the process list and send it the TERM signal.
+        loggerinst.info("Skipping RHSM service shutdown on CentOS 6.")
```

Comment:
```suggestion
loggerinst.info("Skipping RHSM service shutdown on {0} {1}.".format(system_info.name, system_info.version.major))
```
The message would be confusing on Oracle Linux 6. But it's nothing major.